packages/manager/modules/exchange/src/disclaimer/update/disclaimer-update.controller.js

```diff
@@ -25,6 +25,9 @@ export default class ExchangeUpdateDisclaimerCtrl {
   loadOptions() {
     this.loadingData = true;
 
+    // Remove tabindex from modal as it clashes with ckeditor modal
+    document.getElementsByClassName('modal')[0].removeAttribute('tabindex');
```

Comment:
it's kind of a dirty fix, .modal class refer to which element, the ckeditor one or the bootstrap one ?
maybe you can scope the query by className a little more narrower, i didn't check but something like :
```js
document.getElementsByClassName('.ckeditor-modal > .modal'); // just an example
```
also please use angularJS api to query elements : https://docs.angularjs.org/api/ng/service/$document

Reply:
I cannot find a better fix for this. I cannot add class as the bootstrap modal is outside the component.
I have used angularJS api to query the modal

Reply:
@frenautvh is this okay ?